iree/test/microbenchmarks/BUILD

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Tests compilation of microbenchmarks.
+
+load("//build_tools/bazel:enforce_glob.bzl", "enforce_glob")
+load("//build_tools/bazel:iree_lit_test.bzl", "iree_lit_test_suite")
+
+package(
+    default_visibility = ["//visibility:public"],
+    features = ["layering_check"],
+    licenses = ["notice"],  # Apache 2.0
+)
+
+iree_lit_test_suite(
+    name = "lit",
+    size = "medium",
+    srcs = enforce_glob(
+        [
+            "linalg_transpose.mlir",
+        ],
+        include =
+            ["*.mlir"],
```

Comment:
We have to exclude some mlir files. `include + exclude` is expected to be the list you passed above.